src/index.tsx

```diff
@@ -8,6 +8,7 @@ import { BlockNumberProvider } from 'lib/hooks/useBlockNumber'
 import { MulticallUpdater } from 'lib/state/multicall'
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
+import { QueryClient, QueryClientProvider } from 'react-query'
```

Comment:
What is this meant to do? It is redundant with existing code, and the `react-query` package has been deprecated. Can it be removed?

I'm most worried that this might interact poorly with our existing query parser.